integration/dynamodb_test.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/containous/traefik/integration/utils"
+	"github.com/containous/traefik/types"
+	"github.com/go-check/check"
+	checker "github.com/vdemeester/shakers"
+)
+
+type DynamoDBSuite struct {
+	BaseSuite
+}
+
+type DynamoDBBackendItem struct {
+	ID      string        `dynamodbav:"id"`
+	Name    string        `dynamodbav:"name"`
+	Backend types.Backend `dynamodbav:"backend"`
+}
+
+type DynamoDBFrontendItem struct {
+	ID       string         `dynamodbav:"id"`
+	Name     string         `dynamodbav:"name"`
+	Frontend types.Frontend `dynamodbav:"frontend"`
+}
```

Comment:
My understanding is that if you embed the struct, it should inherit that tags as well; functionally, it shouldn't be different from copying the struct.

So this is what I have in mind:

```go
type DynamoDBItem struct {
  ID      string        `dynamodbav:"id"`
  Name    string        `dynamodbav:"name"`
}

type DynamoDBBackendItem struct {
	DynamoDBItem
	Backend types.Backend `dynamodbav:"backend"`
}

type DynamoDBFrontendItem struct {
	DynamoDBItem
	Frontend types.Frontend `dynamodbav:"frontend"`
}
```

I'll leave it up to you if you want to give the embedded approach a try or stick to what we have right now. It's not critical for me.